ziti-router/subcmd/run.go

```diff
@@ -36,7 +36,10 @@ import (
 	"syscall"
 )
 
```

Comment:
This doesn't look like it's used anywhere

Reply:
It is read from the flag meta data. See Line 107

Reply:
The flag data is passed through the config machinery in fabric/router/config.go to edge/router